package.json

```diff
@@ -84,6 +84,7 @@
     "styled-components": "^5.2.0",
     "ts-jest": "^26.5.6",
     "ts-loader": "^9.1.2",
```

Comment:
missing yalc dependency as well

Reply:
We are installing it as a global dependency on our system, per the instructions in the documentation. Do we really need to include it as a dependency inside the project? 

Since we use `yarn global add yalc`, we can just use `yalc` in every project on our system.